README.md

```diff
@@ -29,10 +43,38 @@ gitr exec --command='git commit -m "Add latest changes remotely"' --working-dir=
 gitr exec --command='git push' --working-dir="/path/to/git/repo/in/remote/server" --host-address="localhost:2183"
 ```
 
+### Commands Usage
 
-### Exec Usage
+## Serve
 
 ```bash
+git serve [flags]
+
+Starts the git remote server
+
+Example:.
+gitr serve --port=2183
+
+Usage:
+  gitremote serve [flags]
+
+Flags:
+  -a, --allowed-commands stringSlice      The list of allowed git commands
+  -d, --allowed-directories stringSlice   The list of allowed git executable directories
+  -c, --cert-file string                  The TLS cert file (default "server.pem")
+  -k, --key-file string                   The TLS key file (default "server.key")
+  -p, --port int                          Port for the rpc service (default 2183)
+  -t, --tls                               Connection uses TLS if true, else plain TCP
+
+Global Flags:
+      --config string   config file (default is $HOME/.gitremote.yaml)
```

Comment:
Yes, it is useful, the readme shows an example on how to use the file to configure the server